xarray/tests/test_parallelcompat.py

```diff
@@ -167,9 +169,11 @@ def test_get_dask_if_installed(self) -> None:
         chunkmanager = guess_chunkmanager(None)
         assert isinstance(chunkmanager, DaskManager)
 
-    @pytest.mark.skipif(has_dask, reason="requires dask not to be installed")
-    def test_dont_get_dask_if_not_installed(self) -> None:
-        with pytest.raises(ValueError, match="unrecognized chunk manager dask"):
+    def test_no_chunk_manager_available(self, monkeypatch) -> None:
+        monkeypatch.setattr(
+            "xarray.namedarray.parallelcompat.list_chunkmanagers", dict
```

Comment:
I'm actually not sure whether using `dict` is an improvement over `lambda: {}` in this case...

With the latter it is pretty clear that we have a function that returns an empty `dict`, while the former is more along the lines of "create a new `dict`", which as far as intent goes is not quite the same (the result is the same, though). Sure, maybe `dict` is a tiny bit faster, but we only call this *once*, while running the tests.